CONTRIBUTING.md

```diff
@@ -288,7 +230,7 @@ This assumes that you are on a branch where the versions in all package.json fil
 Doing a stable release after a prerelease requires you to use:
 
 ```
-NPM_CONFIG_OTP={TOKEN} lerna publish --force-publish --conventional-graduate && lerna bootstrap
+NPM_CONFIG_OTP={TOKEN} lerna publish --force-publish --conventional-graduate
```

Comment:
TBD: Do we need do add `npm i` instead? 🤔

Reply:
Uhh, not sure. The `lerna bootstrap` was there because `lerna publish` left the local repository in a weird state where the leaf packages would have their node_modules folder removed (if I remember correctly). The additional `lerna bootstrap` fixed that then. This could be totally different after upgrading from lerna 2 to 7. Suggestion: Let's not add `npm i` now and let's see when we do the next major release. Also, we wanted to create a CI job for that anyway, and then it doesn't matter in which state the local folder is after the release.